libexecutive/Executive.cpp

```diff
@@ -102,6 +106,8 @@ bool Executive::execute()
         if (m_t.gas() < (u256)m_baseGasRequired)
         {
             m_excepted = TransactionException::OutOfGasBase;
+            m_exceptionReason << LOG_KV(
+                "reason", "The gas required by deploying this contract is more than sender given");
```

Comment:
log with `m_baseGasRequired` and `m_t.gas()`

Reply:
done